src/test/resources/github-app-private-key-tests.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Just to be sure, I assume this is a dummy key that you generated purely for testing, yes? :)

Reply:
Yes :), thanks for asking